automated install/basic-install.sh

```diff
@@ -109,14 +110,13 @@ c=$(( c < 70 ? 70 : c ))
 skipSpaceCheck=false
 reconfigure=false
 runUnattended=false
-INSTALL_WEB_SERVER=true
 # Check arguments for the undocumented flags
 for var in "$@"; do
     case "$var" in
         "--reconfigure" ) reconfigure=true;;
         "--i_do_not_follow_recommendations" ) skipSpaceCheck=true;;
         "--unattended" ) runUnattended=true;;
-        "--disable-install-webserver" ) INSTALL_WEB_SERVER=false;;
+        "--disable-install-webserver" ) INSTALL_WEB_SERVER=false INSTALL_WEB_INTERFACE=false;;
```

Comment:
As I mentioned in the other PR, this breaks most use cases of `--disable-install-webserver`. It is not used to skip the web interface install but to allow other webservers or things like Lighttpd with PHP-FPM (instead of PHP-CGI).

But the other way round it makes sense: If the web interface is not wanted (extremely rare case), then the web server is definitely not required. Related PR: #3225

Reply:
See my comments on https://github.com/pi-hole/pi-hole/pull/3590 (Which, looking at it, includes the same changesets...)

Reply:
That makes sense.  I'll have to make a change to the validate_setupVars function also.